tools/tests/py2fgen/test_codegen.py

```diff
@@ -249,7 +249,6 @@ def test_python_wrapper(dummy_plugin):
 import cupy as cp
 from numpy.typing import NDArray
 from gt4py.next.iterator.embedded import np_as_located_field
```

Comment:
Is this necessary here? `import gt4py.next as gtx`

Reply:
same as above